module/Application/src/Application/Entity/User.php

```diff
@@ -264,6 +264,15 @@ public function isMemberOf($organization) {
 		}
 		return $this->memberships->containsKey($key);
 	}
+	
+	public function getMembershipOf($organizationId){
+		foreach ($this->memberships as $m){
```

Comment:
`$this->memberships` è una mappa che ha per chiave l'id dell'organizzazione, quindi puoi accedervi direttamente